components/grid/grid.css

```diff
@@ -0,0 +1,55 @@
+.grid-two-equal-container {
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-areas: "container1 container1 container2 container2";
+}
+.grid-three-equal-container {
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
+  grid-template-areas: "container1 container1 container2 container2 container3 container3";
+}
+.grid-four-equal-container {
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-areas: "container1 container2 container3 container4";
+}
+
+.grid-wrapper {
+  height: 14rem;
+}
+
+.grid-container-1 {
+  grid-area: container1;
+  box-sizing: border-box;
```

Comment:
You can declare all elements to border box at the start of css , just a suggestion btw

Reply:
Thanks for the suggestion.